audformat/core/scheme.py

```diff
@@ -211,6 +213,102 @@ def to_pandas_dtype(self) -> typing.Union[
             return 'timedelta64[ns]'
         return self.dtype
 
+    def replace_labels(
+            self,
+            labels: typing.Union[dict, list],
+    ):
+        r"""Replace labels.
+
+        If scheme is part of a :class:`audformat.Database`
+        the dtype of :class:`audformat.Column`s that reference the scheme
+        will be updated. Removed labels are set to ``NaN``.
+
+        Args:
+            labels: new labels
+
+        Raises:
+            ValueError: if scheme does not define labels
+            ValueError: if dtype of new labels does not match dtype of
+                scheme
+
+        Example:
+            >>> speaker = Scheme(
+            ...     labels={
+            ...         0: {'gender': 'female'},
+            ...         1: {'gender': 'male'},
+            ...     }
+            ... )
+            >>> speaker
+            dtype: int
+            labels:
+              0: {gender: female}
+              1: {gender: male}
+            >>> speaker.replace_labels(
+            ...     labels={
```

Comment:
I think is fine to be slightly less verbose here:
```suggestion
            ...     {
```